packages/sdk/src/interface.ts

```diff
@@ -0,0 +1,43 @@
+
+export interface JobModel {
```

Comment:
I think we can add a TODO comment which shares these interfaces between daemon and sdk.